component/src/main/java/me/androidbox/component/agenda/AlarmReminder.kt

```diff
@@ -44,8 +51,20 @@ fun AlarmReminder(
                     color = MaterialTheme.colorScheme.agendaBodyTextColor)
             }
 
-            Row {
-                Icon(painter = painterResource(id = R.drawable.forward_arrow), contentDescription = "forward arrow")
+            if(isEditMode) {
+                Row {
+                    if(isEditMode) {
+                        IconButton(onClick = {
+                            onReminderClicked()
+                        }) {
+                            Icon(
+                                painter = painterResource(id = R.drawable.forward_arrow),
+                                contentDescription = stringResource(R.string.forward_arrow),
+                                tint = Color.Black
+                            )
+                        }
+                    }
```

Comment:
This check is redundant due to the outer check

Reply:
@philipplackner Thanks for spotting removed